### PR TITLE
Use `graphql-got`

### PR DESCRIPTION
**Pull request by kevva**
Convenient and handles GraphQL errors automatically.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -48,15 +48,14 @@ const query = `
...
 
 async function fetchRepos() {
-	const {body} = await got.post('https://api.github.com/graphql', {
-		json: true,
-		body: {query},
+	const {body} = await graphqlGot('https://api.github.com/graphql', {
+		query,
 		headers: {
 			authorization: `bearer ${token}`
 		}
```
Comment (sindresorhus) [inline index.js]: Should it handle `token` too? Like `gh-got`? I assume most GraphQL endpoints require a token.

File: index.js [hunk 2]
```diff
@@ -48,15 +48,14 @@ const query = `
 let responseText = '[]';
 
 async function fetchRepos() {
-	const {body} = await got.post('https://api.github.com/graphql', {
-		json: true,
-		body: {query},
+	const {body} = await graphqlGot('https://api.github.com/graphql', {
```
Comment (sindresorhus) [inline index.js]: Maybe also support protocol-less in `graphql-got`? So we could just do `api.github.com/graphql`. Of course prepending HTTPS, not HTTP.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (kevva) [reply index.js]: Good idea!

File: index.js (hunk 2 above)
Comment (kevva) [reply index.js]: Fixed.

File: index.js (hunk 1 above)
Comment (kevva) [reply index.js]: Feature added now :).

**Review by sindresorhus — APPROVED** [review]

**Comment by sindresorhus** [comment]
Great! I love how simple the code is now after this and your other PR 🙌

**Pull request merged**